bigquery/bigquery_query/main.tf

```diff
@@ -0,0 +1,26 @@
+/**
+* Copyright 2025 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+# [START bigquery_query]
+resource "google_bigquery_job" "my_query_job" {
```

Comment:
I think this is missing a google_bigquery_table" resource because when you run this it needs to output somewhere or you lose the output. 

I took a look at the example usage and they do that in the version they have as the basic usage

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_job

and then the query has the destination_table defined.

Reply:
I plan to do that in a separate example. This one is just for running the query without saving the results to a table. Users can retrieve the results from a one-off job like this from  BQ jobs history.